### ISSUE TITLE
[Bug] rcb-toggle-voice event does not work

### ISSUE THREAD

**Issue by zikpefu** [tjtanjin/react-chatbotify#121]
**Bug Description**
I'm trying to test the rcb-toggle-voice event and I am not getting any of the console logs to trigger.

**Steps To Reproduce**

This is the code that I used to test the voice listener:
```
const MyChatBot = () => {

  const settings = {
    general: {
      embedded: true
    },
    chatHistory: {
      storageKey: "test"
    },
    event: {
      // opt-in to rcb user submit text event
      rcbToggleVoice: true,
    },
    voice: { disabled: false }
  };

  React.useEffect(() => {
    const test = (event) => {
      console.log('test worked');
    };

    window.addEventListener("rcb-toggle-voice", test);
    return () => {
      window.removeEventListener("rcb-toggle-voice", test);
    };
  }, []);

  // example flow
  const flow = {
    start: {
      message: "Hello there! Try saying hello to me!",
      path: "end"
    },
    end: {
      message: "Keep trying!",
      path: "start"
    }
  };

  return (
    <ChatBot settings={settings} flow={flow} />
  );
};

render(<MyChatBot />);
```

Steps to reproduce the bug behavior:
1. Go to the Chatbotify interactive editor
2. Paste the code above
3. Open the browser console
4. Record your voice and observe no event triggers

**Expected behavior**
Log of "test worked" will show

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Web
 - Browser: Chrome

**Additional context**

**Comment by tjtanjin** [comment]
Hey @zikpefu, thank you for raising this up. Embarrassingly, this bug was caused by a typo where instead of `rcb-toggle-voice`, I wrote `rcb-voice-toggle` 😅

The good news is we know the source of the bug and it is trivial to fix: https://github.com/tjtanjin/react-chatbotify/blob/main/src/constants/RcbEvent.ts

The not-so-good news is I'm overseas right now and can only tend to this when I get back, which is the middle of next week 🥹

If you're in urgent need for this, you can clone the repository, apply the fix yourself and run `npm pack` to build a fixed version. Otherwise, you can wait for the patch next week 😊

**Comment by tjtanjin** [comment]
Hey @zikpefu, this is now resolved in **v2.0.0-beta.13** 😊

**Comment by zikpefu** [comment]
Thank you!
